src/main/java/io/mosip/mimoto/service/impl/IssuersServiceImpl.java

```diff
@@ -113,7 +110,7 @@ public CredentialIssuerWellKnownResponse getIssuerWellknown(String issuerId) thr
                 .filter(issuer -> issuer.getCredential_issuer().equals(issuerId))
                 .findFirst()
                 .map(issuerDTO -> {
-                    String wellknownResponse = restApiClient.getApi(issuerDTO.getWellKnownEndpoint(), String.class);
+                    String wellknownResponse = restApiClient.getApi(issuerDTO.getWellknown_endpoint(), String.class);
```

Comment:
getters in DTO should be in camel case only

Reply:
We are using snake case only for getters and setters in all places. 
To maintain consistency keeping snake case only and we will convert the complete getter and setters to camelcase later